Framework/Algorithms/test/CropWorkspaceTest.h

```diff
@@ -383,9 +391,32 @@ class CropWorkspaceTest : public CxxTest::TestSuite {
     AnalysisDataService::Instance().remove(wsName);
     delete alg;
   }
+  // Test that a non-ragged 2DWorkspace will keep the property where all XHistograms share a pointer
+  // after CropWorkspace
+  void testCropDoesNotCopyXData() {
+    Workspace2D_sptr inputWS = WorkspaceCreationHelper::create2DWorkspace(10, 10);
 
-private:
-  CropWorkspace crop;
+    const auto in_x0_address = &inputWS->x(0);
+    const auto in_x2_address = &inputWS->x(2);
+
+    TS_ASSERT_EQUALS(in_x0_address, in_x2_address);
```

Comment:
```suggestion
    TS_ASSERT_EQUALS(&inputWS->x(0), &inputWS->x(2));
```

Reply:
I like to explicitly name the variables here so that what is happening can be understood quickly without the need for a comment.